main.go

```diff
@@ -34,6 +36,8 @@ func main() {
 
 	l.Log.Info("SuperKey Worker started.")
 
+	initMetrics()
```

Comment:
Can we move this method call above the log messages there? Right below the `l.InitLogger(conf)` call.